.gitignore

```diff
@@ -1,76 +0,0 @@
-### ATTENTION ###
```

Comment:
you should only delete what you added, not the entire file :)

Reply:
the file is restored ;)